centreon/www/widgets/src/centreon-widget-resourcestable/src/Listing/Listing.tsx

```diff
@@ -136,7 +141,7 @@ const Listing = ({
             isOpenTicketEnabled={isOpenTicketEnabled}
           />
         }
-        actionsBarMemoProps={[displayType, hasMetaService,isOpenTicketEnabled]}
+        actionsBarMemoProps={[displayType, hasMetaService, isOpenTicketEnabled]}
```

Comment:
```suggestion
        actionsBarMemoProps={[displayType, hasMetaService, isOpenTicketEnabled, isOnPublicPage]}
```

Related to my previous comment